MovieApp/MovieApp/App/Features/Movies/ViewControllers/SearchMoviesViewController/MoviesSearchViewController.swift

```diff
@@ -19,16 +22,38 @@ class MoviesSearchViewController: UIViewController {
     override func viewDidLoad() {
         super.viewDidLoad()
         
+        buildViews()
+        styleNavigationController()
         presenter.setMoviesViewDelegate(moviesViewDelegate: self)
         presenter.fetchMovies()
-        buildViews()
+        searchBarStackView.setDelegate(delegate: self)
+        searchBarStackView.cancelButton.addTarget(self, action: #selector(popViewController), for: .touchUpInside)
```

Comment:
everything connected to views and setuping them should be called before any fetch